### PR TITLE
Update test README.md

### PR DESCRIPTION
**Pull request by renfeiw**
- updated test tag part
- updated test target part

Signed-off-by: Renfei Wang <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
reviewers: @smlambert @llxia

File: test/README.md [hunk 1]
```diff
@@ -94,6 +94,16 @@ make test
...
+    - A test can be tagged with elements to describe its attribute:
+      - level:   [sanity|extended] (extended default value)
+      - group:   [functional|openjdk|external|perf|jck|system] (required 
+                 to provide one group per test.)
+      - impl:    [openj9|hotspot] (filter test based on exported JAVA_IMPL 
+                 value; openj9 is the default value)
+      - subset:  [SE80|SE90] (filter test based on exported JAVA_VERSION 
+                 value; a test can be tagged with SE80 and SE90 at the 
+                 same time; default to all subsets)
+
```
Comment (llxia) [inline test/README.md]: I would say something like `a test can be tagged with multiple subsets`

File: test/README.md [hunk 2]
```diff
@@ -94,6 +94,16 @@ make test
     examples with your own code, update the build.xml and playlist.xml
     files to match your new Test class names.
 
+    - A test can be tagged with elements to describe its attribute:
```
Comment (llxia) [inline test/README.md]: A test can be tagged with `following` elements:

**Comment by llxia** [comment]
Overall looks good to me. :+1:

File: test/README.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ make test
     JAVA_BIN=<path to JDK bin directory that you wish to test>
     SPEC=[linux_x86-64|linux_x86-64_cmprssptrs|...] platform_on_which_to_test
     JAVA_VERSION=[SE80|SE90] (SE90 default value)
```
Comment (smlambert) [inline test/README.md]: We already have the need for SE100 (at Adopt), so I would add it in the README

File: test/README.md [hunk 4]
```diff
@@ -92,7 +92,18 @@ make test
...
-    files to match your new Test class names.
+    files to match your new Test class names. The playlist.xml format 
+    is defined in TestConfig/playlist.xsd.
+
+    - A test can be tagged with following elements:
+      - level:   [sanity|extended] (extended default value)
+      - group:   [functional|openjdk|external|perf|jck|system] (required 
+                 to provide one group per test.)
+      - impl:    [openj9|hotspot] (filter test based on exported JAVA_IMPL 
+                 value; openj9 is the default value)
```
Comment (smlambert) [inline test/README.md]: I had thought that the lack of a tag for <impl> would mean that the test can be run against both openj9 or hotspot implementations.  Presumably, if openj9 is the default, it means that we have to tag all systemtests and openjdk regression tests at Adopt with both hotspot and openj9?

File: test/README.md [hunk 5]
```diff
@@ -92,7 +92,18 @@ make test
...
+      - subset:  [SE80|SE90] (filter test based on exported JAVA_VERSION
```
Comment (smlambert) [inline test/README.md]: SE100 too

File: test/README.md (hunk 4 above)
Comment (renfeiw) [reply test/README.md]: Yes, it's both. I made a mistake, updating the readme now.

**Comment by renfeiw** [comment]
In addition to SE100, I also added Panama and Valhalla in readme.

**Comment by renfeiw** [comment]
I added Panama and Valhalla, because the two projects also live in the open. I can remove them from readme, let me know.

**Comment by smlambert** [comment]
We will leave them in the doc

**Pull request merged**